security/handler/security-handler-saml/src/main/java/org/codice/ddf/security/handler/saml/SAMLAssertionHandler.java

```diff
@@ -13,13 +13,15 @@
  */
 package org.codice.ddf.security.handler.saml;
 
+import java.io.ByteArrayInputStream; //
```

Comment:
Haha, whoops. I was using those to mark lines when I was trying to fix some checkstyle problems with the import order. I'll remove those.